src/cio_http_server.c

```diff
@@ -109,16 +110,15 @@ static const char *get_response_statusline(enum cio_http_status_code status_code
 {
 	switch (status_code) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a "default" case to this switch statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=c%3ASwitchWithoutDefault)